prac_06/languages.py

```diff
@@ -0,0 +1,15 @@
+"""
+Programming Languages
+estimated: 30 minutes
+actual: 16 minutes for main code, 22 including docstrings etc.
+"""
+from programming_language import ProgrammingLanguage
```

Comment:
Should this be:
from prac_06.programming_language import ProgrammingLanguage
?

Reply:
good point! Completely forgot about that